src/mo_frontend/parser.mly

```diff
@@ -586,6 +586,10 @@ exp_post(B) :
     { CallE(e1, inst, e2) @? at $sloc }
   | e1=exp_post(B) BANG
     { BangE(e1) @? at $sloc }
+  | LPAR SYSTEM e1=exp_post(B) DOT x=id RPAR
```

Comment:
This can be used for all nested `system` module bindings, right? I.e. not only actor classes...

Reply:
Yep, should we need that. But a user can't define system modules unless we add some hacks.